Dynatrace/dynatrace-operator
# Moves go build tags to a common location

# Description

 We had inconsistent use of our go build tags, so this moves it into a shell script that is used all over the codebase to get the correct build tags.

## How can this be tested?
you can run `./hack/build/create_go_build_tags.sh true` to check the tags (the `true` arg is for if we need the e2e tag or not)

## Checklist
